src/axolotl/utils/data/utils.py

```
@@ -69,39 +69,41 @@ def sha256(to_hash: str, encoding: str = "utf-8") -> str:
+        hashes = hashed["row_hash"]
+
+    if other_dataset is not None:
+        other_hashed = other_dataset.map(
+            compute_row_hash,
+            batched=False,
+            num_proc=num_proc,
+            remove_columns=[],
+        )
+        other_hashes = set(other_hashed["row_hash"])
+
+    for idx, hash in enumerate(hashes):
```

Pylint (3.3.7)

[warning] Redefining built-in 'hash'

(W0622)